### PR TITLE
[CFM-960-fix] Allow flagging of content type 'news'.

### PR DESCRIPTION
**Pull request by anvmn**
https://issuetracker.amplexor.com/jira/browse/CFM-960

### CODE REVIEW

File: project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.features.inc [hunk 1]
```diff
@@ -280,6 +285,7 @@ function c4m_features_og_group_dashboard_flag_default_flags() {
       'activity_group' => 0,
       'block_list' => 0,
       'group_dashboard' => 0,
+      'pinned_content' => 0,
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.features.inc]: why do we need the pinned content in here ?

File: project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.features.inc [hunk 2]
```diff
@@ -179,18 +182,19 @@ function c4m_features_og_group_dashboard_flag_default_flags() {
...
     'show_on_form' => 0,
     'access_author' => '',
     'show_contextual_link' => 0,
     'i18n' => 0,
-    'api_version' => 3,
     'module' => 'c4m_features_og_group_dashboard',
     'locked' => array(
       0 => 'name',
     ),
+    'api_version' => 3,
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.features.inc]: revert this change, it seems like only the order has been changed

File: project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.info [hunk 3]
```diff
@@ -30,12 +30,12 @@ features[flag][] = subscribe_c4m_follow_node
 features[flag][] = subscribe_c4m_follow_term
 features[og_features_permission][] = node:group:invite any user
 features[og_features_permission][] = node:project:invite any user
+features[user_permission][] = flag subscribe_c4m_follow_comment
```
Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.info]: @ordavidil 
I recreated the feature. It changed the order to alphabetic.
That's why we got this switch.
We'll have to revert it every time feature is created. Why not change it to default?

Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.info]: I want to believe that whoever export this feature didn't do it manually, hence it seems like a mismatch between machines configuration and it is a never ending game.
Please revert this to decrease the diff of this PR, there is no point to commit this now.

Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_group_dashboard/c4m_features_og_group_dashboard.info]: @ordavidil Done
